server.php

```diff
@@ -14,7 +14,7 @@
 // This file allows us to emulate Apache's "mod_rewrite" functionality from the
 // built-in PHP web server. This provides a convenient way to test a Laravel
 // application without having installed a "real" web server software here.
-if ($uri !== '/' && file_exists(__DIR__.'/public'.$uri)) { //TODO revert this
+if ($uri !== '/' && file_exists(__DIR__.'/public/ghap'.$uri)) { //$uri !== '/' && file_exists(__DIR__.'/public'.$uri
```

Comment:
This directory structure here actually reflects the old structure, where the site root is at `public/ghap` directory. This has been changed to `public` now. So please change this to the default `public` directory.